### PR TITLE
[MOSIP-27671]Added packet sync for packet bulk upload feature

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: admin/admin-service/src/main/java/io/mosip/admin/AdminBootApplication.java [hunk 1]
```diff
@@ -8,7 +8,8 @@
 import org.springframework.scheduling.annotation.EnableAsync;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
 
-@SpringBootApplication(scanBasePackages = { "io.mosip.admin.*", "${mosip.auth.adapter.impl.basepackage}" })
+@SpringBootApplication(scanBasePackages = { "io.mosip.admin.*", "io.mosip.commons.*", 
+		"${mosip.auth.adapter.impl.basepackage}","io.mosip.commons.packet.spi.*" })
```
Comment (ase-101) [inline admin/admin-service/src/main/java/io/mosip/admin/AdminBootApplication.java:12]: "io.mosip.commons.*" is already present, do we really require "io.mosip.commons.packet.spi.*" ?

File: admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/batch/PacketJobResultListener.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	            Optional<Integer> readResult = jobExecution.getStepExecutions().stream().map(StepExecution::getCommitCount).findFirst();
+	            int readCount = readResult.isPresent() ? readResult.get() : 0;
+	            int commitCount = commitResult.isPresent() ? commitResult.get() : 0;
+	            String message = String.format(STATUS_MESSAGE, commitCount,
+	                    jobExecution.getStatus().toString(), failures.isEmpty() ? "0 Errors" : failures.toString());
+	            auditUtil.setAuditRequestDto(EventEnum.getEventEnumWithValue(EventEnum.BULKDATA_UPLOAD_COMPLETED,
+	                            jobId + " --> " + message));
+	            bulkUploadTranscationRepository.updateBulkUploadTransactionPacket(jobId, jobExecution.getExitStatus().getExitCode(), commitCount, message);
+	        } catch (Exception t) {
+	        	t.printStackTrace();
```
Comment (ase-101) [inline admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/batch/PacketJobResultListener.java:82]: Remove t.printStackTrace();

**Review by ase-101 — COMMENTED** [review]
Pls check comments

File: admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/batch/PacketJobResultListener.java (hunk 2 above)
Comment (balaji-alluru) [reply admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/batch/PacketJobResultListener.java]: removed it as suggested

File: admin/admin-service/src/main/java/io/mosip/admin/AdminBootApplication.java (hunk 1 above)
Comment (balaji-alluru) [reply admin/admin-service/src/main/java/io/mosip/admin/AdminBootApplication.java]: It is not required and removed as suggested

**Review by ase-101 — APPROVED** [review]

**Pull request merged**